src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp

```diff
@@ -201,6 +317,7 @@ class instance_trial_of_the_champion : public InstanceMapScript
                     m_auiEncounter[1] = uiData;
                     if (Creature* pAnnouncer = instance->GetCreature(uiAnnouncerGUID))
                     {
+                        DoUpdateAchievementCriteria(ACHIEVEMENT_CRITERIA_TYPE_BE_SPELL_TARGET, SPELL_ACHIEVEMENT_EADRIC);
```

Comment:
Needs sniffs on this but I assume boss should cast the spell on death, this is a hack